ddtrace/tracer/option.go

```diff
@@ -138,6 +138,21 @@ type config struct {
 
 	// enabled reports whether tracing is enabled.
 	enabled bool
+
+	// postProcessor holds the function used to process finished spans of a trace.
+	// It reports whether the trace should be dropped.
+	postProcessor func([]ReadWriteSpan) bool
+}
+
+// WithPostProcessor enables processing finished spans of a trace by f. Spans
+// can be read from and written to in the processor by using the methods specified by
+// the ReadWriteSpan interface. f should return false if the trace should be dropped.
+// todo: document performance impact and inaccurate stats if client side
```

Comment:
The most we can say today is "Dropped traces and spans will not be accounted for in APM stats." because client-side stats is still a feature flag (not GA).